README.md

```diff
@@ -57,7 +57,9 @@ keymap("n", "<leader>rf", genghis.renameFile)
 keymap("n", "<leader>mf", genghis.moveAndRenameFile)
 keymap("n", "<leader>nf", genghis.createNewFile)
 keymap("n", "<leader>yf", genghis.duplicateFile)
-keymap("n", "<leader>df", function() genghis.trashFile { trashLocation = "your/path" } end) -- default: "$HOME/.Trash".
+-- On *Linux* the `trashCmd` argument defaults to `gio trash`.
+-- On *Mac* the `trashLocation` argument defaults to `$HOME/.Trash`.
```

Comment:
On Windows, too (though I am not sure whether it makes sense there, don't have a Windows device, but nobody complained yet, so ...)